ui/app/ducks/swaps/swaps.js

```diff
@@ -480,7 +479,7 @@ export const signAndSendTransactions = (history, metaMetricsEvent) => {
       available_quotes: getQuotes(state)?.length,
       other_quote_selected: usedQuote.aggregator !== getTopQuote(state)?.aggregator,
       other_quote_selected_source: usedQuote.aggregator === getTopQuote(state)?.aggregator ? '' : usedQuote.aggregator,
-      gas_fees: formatCurrency(gasEstimateTotalInEth, 'usd')?.slice(1),
+      gas_fees: gasEstimateTotalInUSD,
```

Comment:
Formatting is both unnecessary and diverges from what we want in our analytics, which is an unformatted decimal number